rules-tests/Php82/Rector/New_/FilesystemIteratorSkipDots/Fixture/constant_present.php.inc

```diff
@@ -0,0 +1,15 @@
+<?php
```

Comment:
please rename fixture file name to : `skip_constant_present.php.inc`

Reply:
I renamed the file